build.gradle

```diff
@@ -255,6 +255,33 @@ model {
                 }
                 targetPlatform p.name
             }
+            binaries.all {
+                if (targetPlatform.operatingSystem.macOsX
+                    || targetPlatform.operatingSystem.linux) {
+                    cppCompiler.args "-g"                       // Produce debug output
+                    cppCompiler.args "-pthread"                 // Force nicer threading
+                    cppCompiler.args "-pedantic"                // Disable non-standard things
+                    cppCompiler.args "--std=c++11"              // Enable C++11
+                    cppCompiler.args "-Wall"                    // All warnings
+                    cppCompiler.args "-Wextra"                  // Plus extra
+                    cppCompiler.args "-Wformat=2"               // Check printf format strings
+                    cppCompiler.args "-Werror"                  // Warnings are errors
+                    cppCompiler.args "-Wno-format-nonliteral"   // Allow printf to have dynamic format string
+                    linker.args "-pthread"
```

Comment:
Why do we now need to use `-pthread` which we didn't do before?

Reply:
Thad said we should. According to the crash logs we've always used pthreads, but he said in some weird cases the compiler could decide otherwise, and we don't want that.